sw.js

```diff
@@ -19,16 +19,16 @@ self.addEventListener('fetch', event => {
 });
 
 async function precache() {
-    const cache = await caches.open(VERSON);
+    const cache = await caches.open(VERSION);
     return cache.addAll([
-      '/',
-      '/index.html',
-      '/assets/index.js',
-      '/assets/MediaPlayer.js',
-      '/assets/plugins/AutoPlay.js',
-      '/assets/plugins/AutoPause.js',
-      '/assets/index.css',
-      '/assets/CatVideo.mp4',
+   //   '/',
+  //    '/index.html',
+   //   '/assets/index.js',
+   //   '/assets/MediaPlayer.js',
+   //   '/assets/plugins/AutoPlay.js',
+    //  '/assets/plugins/AutoPause.js',
+   ///  '/assets/index.css',
+   //   '/assets/CatVideo.mp4',
```

Comment:
Quick question: Why do we have all this commented? Does that mean you are no longer using service worker on TypeScript version. @sandragil16

Reply:
yes that's the reason if you check now the files are using ts instead of js

Reply:
I get it now, thanks for clarirfying. 

LGTM!